packages/kbn-securitysolution-endpoint-exceptions-common/README.md

```diff
@@ -0,0 +1,20 @@
+# Security Solution Endpoint Exceptions
+
+The package contains common files for the Endpoint Exceptions feature.
+
+`common` in the name highlights that this package is intended to combine any common entities related to Endpoint Exceptions in this package. E.g. the other `kbn-securitysolution-endpoint-exception-list-*` packages
+content should be moved here while `kbn-securitysolution-io-ts-list-types` package should be
+gone eventually.
+
+## API folder
+
+`api` folder contains OpenAPI schemas for Security Solution Endpoint Exceptions feature. There are automatically generated Zod schemas and TS types for each schemas located in corresponding
+`*.gen.ts` files.
+
+**Please add any Endpoint Exceptions feature related schemas to this package.**
```

Comment:
Nit: this seems like an odd place for a disclaimer. Who is the intended audience for this README, and for this line specifically?

Reply:
I'd say it's not a disclaimer but rather a usage example. The same pattern is used in `kbn-securitysolution-lists-common` and `kbn-securitysolution-exceptions-common` packages. Do you have some suggestion on how to reorganize it?

Reply:
I'm fine with leaving it, it just seemed self-explanatory/noisy to me. 

I suppose that if I were searching for a place to put my new Endpoint Exception schema, and I found this package, and I read 

> `api` folder contains OpenAPI schemas for Security Solution Endpoint Exceptions feature.

but I still wasn't convinced, then this line would serve as explicit confirmation.